ql/src/test/results/clientpositive/llap/alter_change_db_location.q.out

```diff
@@ -11,7 +11,7 @@ PREHOOK: Input: database:newdb
 POSTHOOK: query: describe database extended newDB
 POSTHOOK: type: DESCDATABASE
 POSTHOOK: Input: database:newdb
-newdb		location/in/test		hive_test_user	USER	
+newdb		location/in/test		hive_test_user	USER			
```

Comment:
yes, now the test is printing out white spaces for connector name and remoteDbName. Because this is a NATIVE database, it is expected to contain whitespace.